scripts/pages/factsheets.page.js

```diff
@@ -1,4 +1,8 @@
 import Dropdown from 'components/dropdown.component';
+import 'styles/_base.scss';
+import 'styles/_texts.scss';
+import 'styles/layouts/l-factsheets-place.scss';
+import 'styles/components/factsheets/area-table.scss';
```

Comment:
Remove this import: this file should be imported in its component. Once you call this component, it will import its own styles.